### PR TITLE
thors-serializer 3.0

### PR DESCRIPTION
**Pull request by Loki-Astari**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/t/thors-serializer.rb [hunk 1]
```diff
@@ -25,6 +25,8 @@ class ThorsSerializer < Formula
 
   def install
     ENV["COV"] = "gcov"
+    mkdir -p ~/.ssh
+    ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts
```
Comment (bevanjkay) [inline Formula/t/thors-serializer.rb:29]: Why is this required?

Comment (Loki-Astari) [reply Formula/t/thors-serializer.rb]: The first run failed with:

> Host key verification failed.

When it was pulling down the submodules. I was trying to solve this issue.
I have removed it.
